addon/-private/proxy.ts

```diff
@@ -13,6 +13,12 @@ const CACHE = new TrackedWeakMap<Interpreter<unknown>, Record<string, unknown>>(
 export function reactiveInterpreter(interpreter: Interpreter<unknown>) {
   CACHE.set(interpreter, tracked({}));
 
+  interpreter.subscribe(console.log);
+  /**
+   * TODO: when event.type === 'SPAWN',
+   *  call reactiveInterpreter on state.children[event.id]
+   *  TODO: figure out a way to handle when id isn't passed
```

Comment:
as an added bit of complexity, a spawned thing can be a callback, promise, etc
https://xstate.js.org/docs/guides/actors.html